sh.fhs.KatawaShoujoReEngineered.yaml

```
@@ -0,0 +1,108 @@
+app-id: sh.fhs.KatawaShoujoReEngineered
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: "ksre"
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: "."
+    version: "22.08"
```

this need to match the runtime version
```suggestion
    version: "23.08"
```

---

fxd